src/mumble/TalkingUI.cpp

```diff
@@ -395,9 +395,12 @@ TalkingUIUser *TalkingUI::findOrAddUser(const ClientUser *user) {
 		TalkingUIUser *newUserEntry                = userEntry.get();
 
 		// * 1000 as the setting is in seconds whereas the timer expects milliseconds
-		userEntry->setLifeTime(Global::get().s.iTalkingUI_SilentUserLifeTime * 1000);
+		int silentUserLifeTime = Global::get().s.iTalkingUI_SilentUserLifeTime;
+		if (silentUserLifeTime != -1)
```

Comment:
Good to know, is there a document where all the guidelines are written down? I didn't see it in the Coding Guidelines.
Just fyi we could add "InsertBraces: true" to the clang-format file, but it is only supported with clang-format >= 15 if you are comfortable with having such a "high" version requirement.

Reply:
No, haha. That got me too when I submitted my first pull request. The Mumble style requirements are currently only living in the head of @Krzmbrzl . Maybe this can be written down somewhere at some point :)